include/seqan/align/gaps_anchor.h

```diff
@@ -756,9 +756,9 @@ assignSource(Gaps<TSequence, AnchorGaps<TGapAnchor> > & gaps, TSequence2 const &
  * @endcode
  */
 
-template <typename TSource, typename TGapAnchors, typename TPosition>
+template <typename TSource, typename TGapAnchors, typename TPosition, typename TRightOfViewPos>
 inline TPosition
```

Comment:
`TRightOfViewPos` can be either `RightOfViewPos` or `LeftOfViewPos`, so why not call it something neutral light `TProjDirection` ?